### PR TITLE
[DOCS] Updated document to correspond to the currently released artifacts

### PR DESCRIPTION
**Pull request by nakul02**
@deroneriksson

### CODE REVIEW

**Comment by deroneriksson** [comment]
Thank you @nakul02!

@dusenberrymw Could you look at the python artifact validation and verify?

All other parts LGTM.

**Comment by deroneriksson** [comment]
BTW I wonder if at some point in the future (not on this PR) if we should move some of the version info to _config.yml to possibly simplify its maintenance.

**Comment by nakul02** [comment]
@deroneriksson - yeah, although we end up changing the artifacts themselves sometimes and the rest of the guide might become obsolete, as it did between 0.11 and 0.12.

File: docs/release-process.md [hunk 1]
```diff
@@ -102,86 +102,64 @@ The build artifacts should be downloaded from [https://dist.apache.org/repos/dis
...
-	cd systemml-0.11.0-incubating
-	java -cp ../lib/*:SystemML.jar org.apache.sysml.api.DMLScript -s "print('hello world');"
+	java -cp "./lib/*:systemml-0.13.0-incubating.jar" org.apache.sysml.api.DMLScript -s "print('hello world');"
+	java -cp "./lib/*:SystemML.jar" org.apache.sysml.api.DMLScript -s "print('hello world');"
+	cd ../..
 
 	# verify spark batch mode
-	export SPARK_HOME=/Users/deroneriksson/spark-1.5.1-bin-hadoop2.6
-	$SPARK_HOME/bin/spark-submit SystemML.jar -s "print('hello world');" -exec hybrid_spark
+	export SPARK_HOME=/Users/njindal/spark-2.1.0-bin-hadoop2.7
```
Comment (dusenberrymw) [inline docs/release-process.md]: Can you remove your username here (and below)?  Perhaps just replace with `export SPARK_HOME=~/spark-2.1.0-bin-hadoop2.7`?

Alternatively, just remove the `SPARK_HOME` pieces completely, and just refer to Spark commands directly, assuming they are already on the path (i.e., `spark-submit ....`).

File: docs/release-process.md [hunk 2]
```diff
@@ -102,86 +102,64 @@ The build artifacts should be downloaded from [https://dist.apache.org/repos/dis
...
+	# install numpy, pandas, scipy & set SPARK_HOME
+	pip install numpy
+	pip install pandas
+	pip install scipy
+	export SPARK_HOME=/Users/njindal/spark-2.1.0-bin-hadoop2.7
+	# get into the pyspark prompt
+	cd systemml-0.13.0
+	$SPARK_HOME/bin/pyspark --driver-class-path systemml-java/systemml-0.13.0-incubating.jar
+	# Use this program at the prompt:
+	# import systemml as sml
```
Comment (dusenberrymw) [inline docs/release-process.md]: Uncomment this.

**Comment by nakul02** [comment]
@dusenberrymw - thanks for the suggestions, updated.

**Pull request closed without merging**